Assets/Scripts/Player/Skeleton/Health.cs

```diff
@@ -0,0 +1,284 @@
+                limbHealthInfo[(int)SkeletalSections.overall].brute += dmg;
+                bruteDmgEval(target, tags);
+                break;
+            case 1: 
+                limbHealthInfo[target].burn += dmg;
+                limbHealthInfo[(int)SkeletalSections.overall].burn += dmg;
+                burnDmgEval(target, tags);
+                break;
+            case 2:
+                limbHealthInfo[(int)SkeletalSections.overall].toxic += dmg;
+                break;
+            case 3:
+                limbHealthInfo[(int)SkeletalSections.overall].suffocation += dmg;
+                break;
+        }
+        critEval();
+        dmgDebug();
+    }
+
+    private void bruteDmgEval(int target, Damage.weaponTags[] tags)
+    {
+        for (int i = 0; i < tags.Length; i++)
+        {
+            if ((int)tags[i] == (int)Damage.weaponTags.blunt)
+            {
+                if (!statusBools[target].bruised && Random.Range(Thresholds.thresholds[(int)Entities.Organic].minBruise, Thresholds.thresholds[(int)Entities.Organic].maxBruise) * limbHealthInfo[target].maxLimbHP < limbHealthInfo[target].brute)
+                {
+                    statusBools[target].bruised = true;
+                }
+
+                //Calculates if a limb is crippled
+                if ((!statusBools[target].crippled && !statusBools[target].detached) && Random.Range(Thresholds.thresholds[(int)Entities.Organic].minCrippled, Thresholds.thresholds[(int)Entities.Organic].maxCrippled) * limbHealthInfo[target].maxLimbHP < limbHealthInfo[target].brute)
+                {
+                    statusBools[target].crippled = true;
+                }
+            }
+
+            else if ((int)tags[i] == (int)Damage.weaponTags.stab || (int)tags[i] == (int)Damage.weaponTags.cut)
+            {
+                //Calculates if a limb is bleeding. If true, begin the BloodLoss Coroutine
+                if (!statusBools[target].bleeding && Random.Range(Thresholds.thresholds[(int)Entities.Organic].minBleed, Thresholds.thresholds[(int)Entities.Organic].maxBleed) * limbHealthInfo[target].maxLimbHP < limbHealthInfo[target].brute)
+                {
+                    statusBools[target].bleeding = true;
+                    numBleedingLimbs++;
+                    /*if(numBleedingLimbs == 1)
+                    StartCoroutine("BloodLoss");
+                    StartCoroutine("BloodRegen");
+                    */
+                }
+
+                //Calculates if a limb is crippled
+                if (target != (int)SkeletalSections.torso && !statusBools[target].detached && Random.Range(Thresholds.thresholds[(int)Entities.Organic].minDetached, Thresholds.thresholds[(int)Entities.Organic].maxDetached) * limbHealthInfo[target].maxLimbHP < limbHealthInfo[target].brute)
+                {
+                    statusBools[target].detached = true;
+                    //BodyPart.severMe();
+                }
+            }
+        }
+    }

```

Comment:
Function name should be in PascalCase.

Additionally, it is incredibly hard to follow what is going on here. Nested if statements should be avoided like the plague. Long conditions should be split up and/or moved into their own functions. And things should be named more explicitly! Additionally, the code for crippling checks is repeated. Here is how I would rewrite this section:

```
        private void BruteDamageEvaluation(int target, Damage.weaponTags[] tags)
        {
            for (int i = 0; i < tags.Length; i++)
            {
                switch (tags[i])
                {
                    case (int) Damage.weaponTags.blunt:
                        EvaluateBruise(target);
                        EvaluateCripple(target);
                        break;
                    case (int) Damage.weaponTags.stab:
                    case (int) Damage.weaponTags.cut:
                        EvaluateBleed(target);
                        EvaluateCripple(target);
                        break;
                    default:
                        Debug.LogError($"Unknown weapon tag {tags[i]} when evaluating brute damage status.");
                        break;
                }
            }
        }

        private void EvaluateBruise(int target)
        {
            //Already bruised, ignore.
            if (statusBools[target].bruised)
            {
                return;
            }

            float bruiseAmount = Random.Range(Thresholds.thresholds[(int) Entities.Organic].minBruise,
                Thresholds.thresholds[(int) Entities.Organic].maxBruise);

            //Not enough damage to bruise, ignore.
            if (bruiseAmount * limbHealthInfo[target].maxLimbHP >= limbHealthInfo[target].brute)
            {
                return;
            }

            statusBools[target].bruised = true;
        }

        private void EvaluateCripple(int target)
        {
            //Already crippled, ignore.
            if (statusBools[target].crippled)
            {
                return;
            }

            //Detatched limb can't be crippled, ignore.
            if (statusBools[target].detached)
            {
                return;
            }

            float crippleAmount = Random.Range(Thresholds.thresholds[(int) Entities.Organic].minCrippled,
                Thresholds.thresholds[(int) Entities.Organic].maxCrippled);

            //Not enough damage to cripple, ignore.
            if (crippleAmount * limbHealthInfo[target].maxLimbHP <= limbHealthInfo[target].brute)
            {
                return;
            }
            
            statusBools[target].crippled = true;
        }

        private void EvaluateBleed(int target)
        {
            //Already bleeding, ignore.
            if (statusBools[target].bleeding)
            {
                return;
            }

            float bleedAmount = Random.Range(Thresholds.thresholds[(int) Entities.Organic].minBleed,
                Thresholds.thresholds[(int) Entities.Organic].maxBleed);
            
            //Not enough damage to bleed, ignore.
            if (bleedAmount * limbHealthInfo[target].maxLimbHP <= limbHealthInfo[target].brute)
            {
                return;
            }
            
            statusBools[target].bleeding = true;
            numBleedingLimbs++;
            /*if(numBleedingLimbs == 1)
            StartCoroutine("BloodLoss");
            StartCoroutine("BloodRegen");
            */
        }
```

This can be further improved. The `damageAmount` variables and their associated calculations are nearly identical in all 3 functions, they could be generalized. But that's not important as it is likely to change in the future.